index.js

```diff
@@ -1,4 +1,4 @@
-var PATTERN = /<(svg|img|math)\s+(.*?)src="(.*?)"(.*?)\/?>/gi;
+var PATTERN = /<(svg|img|math)\s+(.*?)src(\s*)=(\s*)"(.*?)"(.*?)\/?>/gi;
```

Comment:
Sometimes when trying to align the HTML to look more readable, one would do things like:

`<svg 
    src    = "path/to/image.svg"
    class = "some-class"
/>
`